src/fragments/lib/datastore/js/data-access/observe-update-snippet.mdx

```diff
@@ -1,11 +1,11 @@
-```ts
+```js
```

Comment:
Updating this since we default to JS in our examples. No additional TS guidance (outside of `setState`) is required.